ios/FluentUI/Navigation/SideBar.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Delegate protocol to handle user events inside the side bar.
+@objc(MSFSideBarDelegate)
+public protocol SideBarDelegate {
+    /// Called after the view representing `TabBarItem` is selected.
+    @objc optional func sideBar(_ sideBar: SideBar, didSelect item: TabBarItem, fromTop: Bool)
+
+    /// Called after the avatar view is tapped in the side bar.
+    @objc optional func sideBar(_ sideBar: SideBar, didActivate avatarView: AvatarView)
+}
+
+/// View for a vertical side bar that can be used for app navigation.
+/// Optimized for horizontal regular + vertical regular size class configuration. Prefer using TabBarView for other size class configurations.
+@objc(MSFSideBar)
+open class SideBar: UIView {
+    /// Delegate to handle user interactions in the side bar.
+    @objc public weak var delegate: SideBarDelegate?
+
+    /// The avatar view that displays above the top tab bar items.
+    @objc open var avatarView: AvatarView? {
+        willSet {
+            if let gestureRecognizer = avatarViewGestureRecognizer {
+                avatarView?.removeGestureRecognizer(gestureRecognizer)
+                avatarViewGestureRecognizer = nil
+            }
+
+            avatarView?.removeFromSuperview()
+        }
+        didSet {
+            if let view = avatarView {
+                view.translatesAutoresizingMaskIntoConstraints = false
+                addSubview(view)
+
+                let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleAvatarViewTapped))
+                avatarViewGestureRecognizer = tapGesture;
+                view.addGestureRecognizer(tapGesture)
+            }
+
+            setupLayoutConstraints()
+        }
+    }
+
+    /// Tab bar iems to display in the top section of the side bar.
+    @objc open var topItems: [TabBarItem] = [] {
```

Comment:
>TabBarItem [](start = 30, length = 10)

I think tabBarItemView is optimized for iPhone. If the "landscapeImage" property is set in the items (which in iPhone usually means a smaller image, because landscape has less vertical space), we'll use that image. On iPad landscape mode doesn't mean less vertical space. Perhaps we should comment that the lanscapeImage should not be set